### ISSUE TITLE
App uses an extremely high amount of mobile data

### ISSUE THREAD

**Issue by drunken-monkey** [austrianredcross/stopp-corona-android#286]
## Description

According to my phone’s mobile data statistics, the Stopp Corona app has used 1.6 GB of mobile data in March, and 800 MB in February. In January, it was just 8 MB, so this seems due to a recent change.

I’m pretty sure this is not intended?

## Reproduction steps

For me, it was just turning the app on now and then (basically, when going into any closed rooms outside my flat). Of course, don’t know if it’s reproducible.

OS: LineageOS 17.1 (weekly builds)
Phone: Samsung A510F
Version of the app: 2.1.0.1179-QA_259

## What did you expect

App should just use a few MB of data each month.

**Comment by TuxCoder** [comment]
yeah , I saw that a few days ago it looks like they always download the full amount of keys,
even a simple diff would be enought and would reduce the size by like 90%.

For example my part of my log:
time | downloaded keys
-------------------------------
15:40 |  288639
15:11 | 285320
14:12 | 282176

from this two newer downloads where it loaded 6463 new keys to the first download.
It downloaded actually  573959  what is 8880% more than needed or you can spare
98.87% of the traffic.

This would not only save a huge amount of bandwith on the phone, where it is expensive and slow.
This would also reduce the server costs by ~99% as traffic is the most compute work.


Please fix this problem, I going to de install the app as nobody is using it and there is no improvement in the last months.

The app used on my phone already 315MB in the last 7 days.

**Comment by adiOdi** [comment]
I have the same problem, but found a temporary "fix": just disallow mobile data access, and let it check through wifi only. This should mitigate most costs. 
(In my case it used a ridiculous 2.8GB in the last month, this is not acceptable!)
The app runs only in the background, I haven't opened it.
